src/backend/app.js

```diff
@@ -20,6 +20,89 @@ app.use(express.json());
 app.use(cors());
 
 router.use("/meals", mealsRouter);
+router.get('/', (req, res) => {
```

Comment:
I'm a bit confused about your path. There are many variables that actually hadn't been used properly. is it `api/meals/` or `meals/`?

Reply:
the path is  './api/meals'